src/service_manager.rs

```diff
@@ -161,7 +159,8 @@ impl ServiceManager {
             launch_command_buffer.push(wide);
         }
 
-        let launch_command = WideCString::from_wide_str(launch_command_buffer).unwrap();
```

Comment:
I don't think it's possible for that call to fail since we double check every single buffer absorbed by `launch_command_buffer`

Reply:
That is true. But the day a panic happens in production a custom message in the panic will really help finding the cause. We have not really cared a lot about this earlier, I have to admit that. But the community is fairly leaning towards always using `expect` over `unwrap`, to easier track down nasty bugs.

It might be that we one day change how we construct `launch_command_buffer` or there might even be a bug in some new version of `from_wide_str` or whatnot that might trigger a panic here.